pkg/metrics/metrics.go

```diff
@@ -448,6 +471,13 @@ func toSeconds(d time.Duration) float64 {
 	return float64(d / time.Second)
 }
 
+// SetBackupCountGauge records the existing number of restores
+func (m *ServerMetrics) SetRestoreCountGauge(size int) {
```

Comment:
I think this function could just increment a counter, rather than take a size. That would be similar to `RegisterRestoreAttempt`.